src/main/java/org/dave/compactmachines3/gui/framework/widgets/Widget.java

```diff
@@ -115,10 +114,10 @@ public String toString() {
 
     public boolean isPosInside(int x, int y) {
         boolean isInsideX = this.getActualX() <= x && x < this.getActualX() + this.width;
-        //Logz.info("[%s] insideX: %d < %d <= %d --> %s", this.id, this.lastDrawX, x, this.lastDrawX + this.lastDrawWidth, this.toString());
+        //Logz.info("[{}] insideX: {} < {} <= {} --> {}", this.id, this.lastDrawX, x, this.lastDrawX + this.lastDrawWidth, this.toString());
```

Comment:
If these are commented out, better to remove the lines since they haven't been used in production for a while

Reply:
Sure. I mainly left them since they were there before. I was using a search and replace function to replace al the string formats with logger formats. I can remove them when I get home.